dotnet/src/Connectors/Connectors.Memory.Redis/RedisVectorRecordStore.cs

```diff
@@ -326,46 +318,25 @@ public async IAsyncEnumerable<string> UpsertBatchAsync(IEnumerable<TRecord> reco
     /// Prefix the key with the collection name if the option is set.
     /// </summary>
     /// <param name="key">The key to prefix.</param>
-    /// <param name="collectionName">The collection name that was provided as part of an operation to override the default or the default if not.</param>
     /// <returns>The updated key if updating is required, otherwise the input key.</returns>
-    private string PrefixKeyIfNeeded(string key, string? collectionName)
+    private string PrefixKeyIfNeeded(string key)
     {
         if (this._options.PrefixCollectionNameToKeyNames)
         {
-            return $"{collectionName}:{key}";
+            return $"{this._collectionName}:{key}";
         }
 
         return key;
     }
 
-    /// <summary>
-    /// Choose the right collection name to use for the operation by using the one provided
-    /// as part of the operation options, or the default one provided at construction time.
-    /// </summary>
-    /// <param name="operationCollectionName">The collection name provided on the operation options.</param>
-    /// <returns>The collection name to use.</returns>
-    private string ChooseCollectionName(string? operationCollectionName)
-    {
-        var collectionName = operationCollectionName ?? this._options.DefaultCollectionName;
-        if (collectionName is null)
-        {
-#pragma warning disable CA2208 // Instantiate argument exceptions correctly
-            throw new ArgumentException("Collection name must be provided in the operation options, since no default was provided at construction time.", "options");
-#pragma warning restore CA2208 // Instantiate argument exceptions correctly
-        }
-
-        return collectionName;
-    }
-
     /// <summary>
     /// Run the given operation and wrap any Redis exceptions with <see cref="VectorStoreOperationException"/>."/>
     /// </summary>
     /// <typeparam name="T">The response type of the operation.</typeparam>
-    /// <param name="collectionName">The name of the collection the operation is being run on.</param>
     /// <param name="operationName">The type of database operation being run.</param>
     /// <param name="operation">The operation to run.</param>
     /// <returns>The result of the operation.</returns>
-    private static async Task<T> RunOperationAsync<T>(string collectionName, string operationName, Func<Task<T>> operation)
+    private async Task<T> RunOperationAsync<T>(string operationName, Func<Task<T>> operation)
```

Comment:
I'm using the collectionName field now.  Since there's only one name per class instance, no need to pass it as a param, but then I need to read it from the field, which means the method cannot be static.  I felt this was nicer than just passing the same field as a parameter everywhere.